server/src/main/java/fr/uga/l3miage/tp4/repositories/ExamRepository.java

```diff
@@ -0,0 +1,15 @@
+package fr.uga.l3miage.tp4.repositories;
+
+import fr.uga.l3miage.tp4.models.ExamEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.swing.*;
+import java.util.List;
+import java.util.Set;
+
+@Repository
+public interface ExamRepository extends JpaRepository<ExamEntity,Long> {
+    List<ExamEntity> findByName(String name);
+    Set<ExamEntity> findAllBySkillsContaining(String skillWord);
```

Comment:
vous êtes pas loin, ici votre requête doit avoir une skill entity à comparer, et pas un string , il fallait regarder le nom du skills
```suggestion
    Set<ExamEntity> findAllBySkillsNameContaining(String skillWord);
```